examples/howto/plot_batch_simulate.py

```diff
@@ -110,21 +110,45 @@ def summary_func(results):
 # backend='dask' if installed
 print("Simulation results:", simulation_results)
 ###############################################################################
-# Extract and Plot Results
-###############################################################################
+# This plot shows an overlay of all smoothed dipole waveforms from the
+# batch simulation. Each line represents a different set of parameters,
+# allowing us to visualize the range of responses across the parameter space.
+
 
-# Extract min and max peaks for plotting
-min_peaks, max_peaks = [], []
-for res in simulation_results:
-    summary_stats = res[0]
-    min_peaks.append(summary_stats['min_peak'])
-    max_peaks.append(summary_stats['max_peak'])
+dpl_waveforms = []
+for data_list in simulation_results['simulated_data']:
+    for data in data_list:
+        window_len = 30
+        dpl_smooth = data['dpl'][0].copy().smooth(window_len)
```

Comment:
```suggestion
        dpl_smooth = data['dpl'][0].copy().smooth(window_len=30)
```

Reply:
Can you add `.copy().smooth(window_len=30)` to the summary statistic function as well?

Right now the values of the min/max peaks don't match what you see in the dipole waveform plot